modules/deployments-comparing-deploymentconfigs.adoc

```diff
@@ -46,26 +69,3 @@ Deployments do not yet support any lifecycle hooks.
 ==== Custom strategies
 
 Deployments do not support user-specified custom deployment strategies yet.
```

Comment:
Not very likely they will ever get supported, so I would remove the `yet` part.

Reply:
Removed, thanks!